furnace.c

```diff
@@ -325,10 +326,24 @@ do_thermocouple_work(furnace_context_t *ctx, bool deadline_met)
   log_stdout_thermocouple(ctx->log_bits, "hot: %u\n", ctx->cur_temp);
 }
 
+static int
+get_basic_info(char* buffer, const size_t size, furnace_context_t *ctx)
```

Comment:
Consider renaming to `format_status()`

Can we also define some constants for max length of the status that user can use to allocate the buffer?